python/pyspark/pandas/internal.py

```diff
@@ -718,8 +927,11 @@ def attach_distributed_sequence_column(
                     sql_ctx,
                 )
                 columns = df.columns
-                return df.selectExpr(
-                    "`{}` as `{}`".format(columns[1], column_name), "`{}`.*".format(columns[0])
+                return (
+                    df.selectExpr(
+                        "`{}` as `{}`".format(columns[1], column_name), "`{}`.*".format(columns[0])
+                    ),
+                    True,
```

Comment:
Why is `force_nullable` `True` here?

Reply:
The `Encoders.tuple` makes object-like elements nullable, which is the struct converted from the original data frame, and `GetStructField.nullable` depends on the child's nullability.

Reply:
Thanks!